examples/lighting-app/efr32/src/gen/call-command-handler.c

```diff
@@ -99,9 +96,28 @@ EmberAfStatus emberAfClusterSpecificCommandParse(EmberAfClusterCommand * cmd)
     {
         switch (cmd->apsFrame->clusterId)
         {
+        case ZCL_BASIC_CLUSTER_ID:
+            result = emberAfBasicClusterServerCommandParse(cmd);
+            break;
+        case ZCL_GROUPS_CLUSTER_ID:
+            result = emberAfGroupsClusterServerCommandParse(cmd);
+            break;
+        case ZCL_IDENTIFY_CLUSTER_ID:
+            result = emberAfIdentifyClusterServerCommandParse(cmd);
+            break;
+        case ZCL_LEVEL_CONTROL_CLUSTER_ID:
+            result = emberAfLevelControlClusterServerCommandParse(cmd);
+            break;
         case ZCL_ON_OFF_CLUSTER_ID:
             result = emberAfOnOffClusterServerCommandParse(cmd);
             break;
+        case ZCL_SCENES_CLUSTER_ID:
```

Comment:
It's the default configuration generated by the ZAP when one add a Light. All of those extra clusters can be disable through ZAP. Since it's application specific, I will let other developers configure their application as they want. Having unused clusters in call-command-handler shouldn't cause any problems.

Reply:
Right, I'm just saying we should match the configuration of the existing app.

As far as the general concept of "unused" clusters in call-command-handler:

1. They affect codesize measurements; it would be good to not change that "accidentally" without a clear "we are changing this" decision.
1. Do they affect behavior when a command is sent to one of the "unused" clusters?  I guess not, because none of the endpoints actually support that cluster, so the command will just get an "invalid cluster" response?

Reply:
The command will get a "Unsupported Command" response if the cluster is "unused" compare to "Invalid cluster" response if we remove the cluster from the configuration.